src/main/java/leaf/cosmere/common/eventHandlers/EntityEventHandler.java

```diff
@@ -67,9 +71,23 @@ public static void onEntityJoinWorldEvent(EntityJoinLevelEvent event)
 			if (eventEntity instanceof Player)
 			{
 				//todo choose based on planet? eg scadrial gets twinborn, roshar gets surgebinding etc?
+				if (!CosmereConfigs.SERVER_CONFIG.ALLOW_METALBORN_CHOICE.get())
 				{
 					//give random power
 					giveEntityStartingManifestation(livingEntity, spiritweb);
+					spiritweb.setHasBeenInitialized();
+				}
+				else
+				{
+					// spiritweb not initialized yet in this case
+					Player player = (Player) eventEntity;
+
+					String command = "/cosmere choose_metalborn_powers ";
+					MutableComponent instructionComponent = Component.literal("To choose powers, use ");
+					instructionComponent.append(Component.literal("§6§n/cosmere choose_metalborn_powers [allomancy] [feruchemy]")
```

Comment:
Let's make this a command that they can actually click on to trigger for them

Reply:
d'oh.... I missed the line literally underneath it calling it a with click event

Never mind!